controller/array_action/array_mediator_xiv.py

```diff
@@ -341,6 +345,11 @@ def create_snapshot(self, volume_id, snapshot_name, pool=None):
             logger.exception(ex)
             raise array_errors.PermissionDeniedError(
                 "create snapshot {0} from volume {1}".format(snapshot_name, volume_id))
+        except xcli_errors.CommandFailedRuntimeError as ex:
+            logger.exception(ex)
+            if MAXIMUM_SNAPSHOTS_REACHED_ERROR in ex.status:
+                raise array_errors.MaximumSnapshotsReached(pool)
```

Comment:
as I [wrote](https://github.com/IBM/ibm-block-csi-driver/pull/349#discussion_r667476177), new exceptions should be added in the mediator interface and wherever applicable in the mediator implementations

Reply:
Done

Reply:
what about svc + ds8k?

Reply:
Currently, we don't catch similar exceptions in SVC/DS8K. Anyway, it's not related to RE status code. Thus, it's not related to this PR.

Reply:
we shouldn't add new exceptions in the interface without adding them wherever applicable in all the mediators.
so please write the ticket number for the "maximum objects reached" handling and remove it from this PR

Reply:
"here" == in this conversation or in the code (as a comment)

Reply:
conversation is sufficient (in this case the code doesn't have any smell that requires a ticket comment)

Reply:
CSI-3183